core/src/com/mygdx/minigolf/model/components/PowerUpAffectable.java

```diff
@@ -0,0 +1,14 @@
+package com.mygdx.minigolf.model.components;
+
+import com.badlogic.ashley.core.Component;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class PowerUpAffectable implements Component {
```

Comment:
@AleksanderWK Skulle denne renames til PowerupTaker nå eller bare i den neste branchen? Kan gjerne gjøre det nå for min del. Blir færre merge conflicts senere.

Reply:
Tenkte å fikse alt med powerups i den nye branchen, men du kan godt fikse det nå.

Reply:
okei tar å endrer på det og merger inn main